scipy/stats/stats.py

```diff
@@ -4153,6 +4154,177 @@ def binary_search(n, n1, n2, side):
     return oddsratio, pvalue
 
 
+def _barnard_exact_compute_gammaln_combination(n):
+    # Compute all log combination of C(n, k)
```

Comment:
you can use a docstring even in private functions.
```suggestion
    """Compute all log combination of C(n, k)"""
```